SPECS/python-conda-package-handling/python-conda-package-handling.spec

```diff
@@ -0,0 +1,125 @@
+%global srcname conda-package-handling
+%global pkgname conda_package_handling
+
+Summary:        Create and extract conda packages of various formats
+Name:           python-%{srcname}
+Version:        1.7.2
+Release:        3%{?dist}
+License:        BSD
+URL:            https://github.com/conda/%{srcname}
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        https://github.com/conda/%{srcname}/archive/%{version}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:  gcc
+BuildRequires:  libarchive-devel
+
+%description
+Create and extract conda packages of various formats.
+
+%package -n python%{python3_pkgversion}-%{srcname}
+Summary:        %{summary}
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+BuildRequires:  python%{python3_pkgversion}-Cython
+BuildRequires:  python%{python3_pkgversion}-six
+BuildRequires:  python%{python3_pkgversion}-tqdm
+Requires:       python%{python3_pkgversion}-six
+Requires:       python%{python3_pkgversion}-tqdm
+%if %{with_check}
+BuildRequires:  python3-pip
+BuildRequires:  python3-atomicwrites
```

Comment:
It looks like this appears both here and under the `pip3 install` line. I think  we could get away with removing the build requirement? Would keep us from breaking the check section if we started using an incompatible version of atomicwrites in our repo.

Reply:
fixed:)